### PR TITLE
grid complete. Switching drivers

### PR DESCRIPTION
**Pull request by Mannyplg**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,26 @@ import Square from './components/Square'
...
+      {squares.map((value, index) => {
+        return(
+        <Square value={value}
+                index={index}
+                />
+        )
+      })}
+
+      </div>    
+    </div>
```
Comment (elysemontano) [inline src/App.js:25]: Watch your indentation in this section.

File: src/components/Square.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 const Square = (props) => {
   return (
-    <div className="square"></div>
+    <div className="square">Anyt??</div>
```
Comment (elysemontano) [inline src/components/Square.js:5]: Did you want the inner HTML here to be hardcoded to "Anyt??" ?

**Review by elysemontano — APPROVED** [review]
Left some comments to take a look at and modify before merging.

File: src/components/Square.js (hunk 2 above)
Comment (Mannyplg) [reply src/components/Square.js:5]: No. Debugging...

**Pull request merged**